<ui version="4.0" >
 <class>myOpenmetaClass</class>
 <widget class="QWidget" name="myOpenmetaClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>245</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Meta Information</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff>./cornell-bigred.png</normaloff>./cornell-bigred.png</iconset>
  </property>
  <widget class="QLabel" name="label_level" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Num. Levels:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_level" >
   <property name="geometry" >
    <rect>
     <x>320</x>
     <y>9</y>
     <width>44</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_level" >
   <property name="geometry" >
    <rect>
     <x>510</x>
     <y>10</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_value" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>60</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Num. Values:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_value" >
   <property name="geometry" >
    <rect>
     <x>280</x>
     <y>90</y>
     <width>44</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_qi" >
   <property name="geometry" >
    <rect>
     <x>22</x>
     <y>145</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Quasi Identifier</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_sa" >
   <property name="geometry" >
    <rect>
     <x>22</x>
     <y>170</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Sensitive Attribute</string>
   </property>
  </widget>
  <widget class="Line" name="line" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2" >
   <property name="geometry" >
    <rect>
     <x>2</x>
     <y>47</y>
     <width>16</width>
     <height>151</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>190</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4" >
   <property name="geometry" >
    <rect>
     <x>212</x>
     <y>47</y>
     <width>16</width>
     <height>151</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>190</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>40</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7" >
   <property name="geometry" >
    <rect>
     <x>594</x>
     <y>47</y>
     <width>16</width>
     <height>151</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8" >
   <property name="geometry" >
    <rect>
     <x>232</x>
     <y>47</y>
     <width>16</width>
     <height>151</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_children" >
   <property name="geometry" >
    <rect>
     <x>350</x>
     <y>60</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>130</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Choose Value</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_value" >
   <property name="geometry" >
    <rect>
     <x>260</x>
     <y>160</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>390</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Choose Hierarchy Level</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>24</x>
     <y>58</y>
     <width>183</width>
     <height>76</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" >
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" >
      <item>
       <widget class="QLabel" name="label_name" >
        <property name="text" >
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_type" >
        <property name="text" >
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_missing" >
        <property name="text" >
         <string>Missing:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QLineEdit" name="lineEdit_name" />
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_type" />
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_missing" />
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>164</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" >
    <item>
     <widget class="QLabel" name="label_attribute" >
      <property name="text" >
       <string>Choose Attributes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_attribute" />
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>280</x>
     <y>210</y>
     <width>286</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" >
    <item>
     <spacer name="horizontalSpacer_2" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>30</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ok" >
      <property name="text" >
       <string>Okay</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancel" >
      <property name="text" >
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
